inst/learnR-chapters/chapter_55_to_60_learning.Rmd

```diff
@@ -704,8 +681,8 @@ Predictive analysis is a challenging task. Hence, it's crucial to evaluate the a
 
 To assess our predictive models' performance, we calculate error rates. The two common ways are:
 
-1. RMSE (Root-mean-square Error)
-2. Accuracy
+1.  RMSE (Root-mean-square Error)
```

Comment:
Any idea about these extra spaces? They are probably not a problem, just curious as to where they came from.

Reply:
quiz just the space before the quiz question